q2_fragment_insertion/citations.bib

```diff
@@ -1,10 +1,12 @@
 @article{SEPP,
   title={Phylogenetic Placement of Exact Amplicon Sequences Improves Associations with Clinical Information},
-	author={Janssen, Stefan and McDonald, Daniel and Gonzalez, Antonio and Navas-Molina, Jose A. and Jiang, Lingjing and Xu, Zhenjiang Zech and Winker, Kevin and Kado, Deborah M. and Orwoll, Eric and Manary, Mark and Mirarab, Siavash and Knight, Rob},
+  author={Janssen, Stefan and McDonald, Daniel and Gonzalez, Antonio and Navas-Molina, Jose A. and Jiang, Lingjing and Xu, Zhenjiang Zech and Winker, Kevin and Kado, Deborah M. and Orwoll, Eric and Manary, Mark and Mirarab, Siavash and Knight, Rob},
   journal={mSystems},
   volume={3},
   number={3},
   year={2018},
   publisher={American Society for Microbiology Journals},
   doi={10.1128/mSystems.00021-18},
 }
+
```

Comment:
# from current docu of HMMER: "There has never been a paper on the HMMER software. The best citation is to the web site, hmmer.org."

@article{hmmer,
    author={Eddy, Sean R.},
    journal={PLOS Computational Biology},
    publisher={Public Library of Science},
    title={Accelerated Profile HMM Searches},
    year={2011},
    month={10},
    volume={7},
    url={https://doi.org/10.1371/journal.pcbi.1002195},
    pages={1-16},
    number={10},
    doi={10.1371/journal.pcbi.1002195}
}
@article{pplacer,
  title={pplacer: linear time maximum-likelihood and Bayesian phylogenetic placement of sequences onto a fixed reference tree},
  author={Matsen, Frederick A and Kodner, Robin B and Armbrust, E Virginia},
  journal={BMC bioinformatics},
  volume={11},
  number={1},
  pages={538},
  year={2010},
  publisher={BioMed Central}
}

Reply:
Perfect, thanks for doing the legwork here!